src/test/java/unit/Group.java

```diff
@@ -61,6 +63,17 @@ void listDescribeUsersReflectCreateDelete() throws IOException {
     // check that the group creator is included in the list and is an admin
     expectListedMemberWithPolicies(name, testUser.email, GroupPolicy.ADMIN);
 
+    // create another group
```

Comment:
In general, when you add a code change, we usually add a corresponding test. Usually the new test fails before the code change, and passes after the code change. In this case, the new test code isn't really related to your code change. The new test code passes before your code change.

I would just delete lines 66-76. @jaycarlton  ?

If you keep this, change `name1` to `name2`. (For the second thing of something, we usually go with `2` instead of `1`.)

Reply:
sorry. Melissa. I did wrong test code and this is a draft review. 

I talked with Jay. He suggests me to use regular expression to test the table by each row. and verify the head and table content. It fails in my local... I am debugging now.